README.md

```diff
@@ -24,13 +24,15 @@ The rest of this document will focus on how this version of XCTest differs from
 
 ## Working on XCTest
 
+### On Linux
+
 XCTest can be built as part of the overall Swift package. When following [the instructions for building Swift](http://www.github.com/apple/swift), pass the `--xctest` option to the build script:
 
 ```sh
 swift/utils/build-script --xctest
 ```
 
-If you want to build just XCTest, use the `build_script.py` script at the root of the project. The `master` version of XCTest must be built with the `master` version of Swift.
+If you want to build just XCTest, use the `build_script.py` script at the root of the project. The `master` version of XCTest must be built with the `master` version of Swift. XCTest has a dependency upon Foundation, so you must have built the `master` version of that as well.
```

Comment:
I specified `master` in the README as a matter of convenience--using the Swift CI we can guarantee that XCTest and Foundation `master` build together. Of course, it might be possible for people to contribute by linking the latest XCTest `master` against an older Foundation snapshot, but it'll be hard for us to keep track of which versions of XCTest works with which versions of Foundation.

I totally agree with your suggestions to make it easier to contribute (and I think it's imperative we keep the barrier to entry as low as possible), but if possible I'd like to leave those as future improvements.